integration/main.go

```diff
@@ -123,7 +123,11 @@ func runGinkgo(ctx context.Context, wg *sync.WaitGroup, summaries chan []string,
 		}
 	}
 
-	args := []string{"--no-color", fmt.Sprintf("--timeout=%s", suiteTimeout), "-tags", "integration", "-v", "--progress"}
+	var args []string
+	if ginkgoArgs := os.Getenv("GINKGO_ARGS"); ginkgoArgs != "" {
+		args = strings.Split(ginkgoArgs, " ")
+	}
+	args = append(args, "--no-color", fmt.Sprintf("--timeout=%s", suiteTimeout), "-tags", "integration", "-v", "--progress")
```

Comment:
```suggestion
	args = append(args, "--no-color", fmt.Sprintf("--timeout=%s", suiteTimeout), "-tags", "integration", "-vv")
```

`--progress` is deprecated 💡

Reply:
I'm aware and we already have an issue for that, but this change is not being introduced in this PR.